src/openforms/logging/logevent.py

```diff
@@ -208,12 +208,17 @@ def registration_success(submission: Submission, plugin):
     )
 
 
-def registration_failure(submission: Submission, error: Exception, plugin=None):
+def registration_failure(
+    submission: Submission, error: Exception, include_in_daily_digest: bool, plugin=None
+):
```

Comment:
```suggestion
def registration_failure(
    submission: Submission, error: Exception, *, plugin: ... = None, include_in_daily_digest: bool
):
```
To avoid [boolean traps](https://adamj.eu/tech/2021/07/10/python-type-hints-how-to-avoid-the-boolean-trap/). Although this makes it a little weird with `plugin` having a default value